config/environments/development.rb

```diff
@@ -127,9 +127,4 @@
   # In order to fix the problem, the following options must be set.
   routes.default_url_options ||= {}
   routes.default_url_options[:script_name]= ''
-
-  # Disable CSRF origin check during development because the HMIS frontend has a different origin
```

Comment:
This is still needed for those of us who are developing the HMIS locally @ttoomey. Maybe you didn't need it because of your docker setup? Anyway, I'm adding it back